.github/workflows/check.yml

```diff
@@ -69,7 +68,7 @@ jobs:
           fi
       
       - name: Static Analyzer Reports Uploading
-        if: steps.analyzer-output.outcome == 'failure'
+        if: ${{ failure() }}
```

Comment:
```suggestion
        if: failure()
```
should be enough, brackets are necessary if you, for example, want to negate it: `if: ${{ !failure() }}`

Reply:
Look at 3d3e878bd080519710493ffc4751c128993c3b47

Reply:
ok, now it makes sense 👍